docs/data-sources/omim.mdx

```diff
@@ -0,0 +1,219 @@
+              }
+            }
+          ],
+          "geneMap": {
+            "sequenceID": 13666,
+            "chromosome": 17,
+            "chromosomeSymbol": "17",
+            "chromosomeSort": 947,
+            "chromosomeLocationStart": 81509970,
+            "chromosomeLocationEnd": 81512798,
+            "transcript": "ENST00000331925.7",
+            "cytoLocation": "17q25.3",
+            "computedCytoLocation": "17q25.3",
+            "mimNumber": 102560,
+            "geneSymbols": "ACTG1, DFNA20, DFNA26, BRWS2",
+            "geneName": "Actin, gamma-1",
+            "mappingMethod": "REa, A, Fd",
+            "confidence": "C",
+            "mouseGeneSymbol": "Actg1",
+            "mouseMgiID": "MGI:87906",
+            "geneInheritance": null,
+            "phenotypeMapList": [
+              {
+                "phenotypeMap": {
+                  "mimNumber": 102560,
+                  "phenotype": "Baraitser-Winter syndrome 2",
+                  "phenotypeMimNumber": 614583,
+                  "phenotypicSeriesNumber": "PS243310",
+                  "phenotypeMappingKey": 3,
+                  "phenotypeInheritance": "Autosomal dominant"
+                }
+              },
+              {
+                "phenotypeMap": {
+                  "mimNumber": 102560,
+                  "phenotype": "Deafness, autosomal dominant 20/26",
+                  "phenotypeMimNumber": 604717,
+                  "phenotypicSeriesNumber": "PS124900",
+                  "phenotypeMappingKey": 3,
+                  "phenotypeInheritance": "Autosomal dominant"
+                }
+              }
+            ]
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+OMIM API can accept almost 20 MIM numbers in one query. To simplify the work to parsing the JSON responses, this gzipped json response file has all the original responses properly concatenated as if they were returned by the server in a single query.
+
+### 3. MimEntries.json.gz.version
+
+The version file is generated automatically after all the data have been downloaded. The date in the local time zone is used to version the data. This means, if the downloader has been run twice a day, the version of downloaded data would be the same, although OMIM website may have an update between the two runs.
+
+## Generate OMIM database
+
+```shell
```

Comment:
I am all for describing the data parsing and filtering process but we should not provide SAUtils commands in this documentation I think.

Reply:
We can talk about this.